game/templates/game/board.html

```
@@ -275,6 +275,8 @@ <h1>Checkora <span class="badge" id="modeBadge">PVP</span></h1>
             </div>
 
             <div style="display: flex; flex-direction: column; gap: 12px;">
+                <button class="btn btn-gold" id="shareResultBtn" style="padding: 12px; font-size: 1rem;">📤 Share Result
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)